shell/key-bindings.bash

```diff
@@ -41,7 +41,7 @@ __fzf_cd__() {
   opts="--height ${FZF_TMUX_HEIGHT:-40%} --bind=ctrl-z:ignore --reverse --walker=dir,follow,hidden --scheme=path ${FZF_DEFAULT_OPTS-} ${FZF_ALT_C_OPTS-} +m"
   dir=$(
     FZF_DEFAULT_COMMAND=${FZF_ALT_C_COMMAND:-} FZF_DEFAULT_OPTS="$opts" $(__fzfcmd)
-  ) && printf 'builtin cd -- %q' "$dir"
+    ) && printf 'builtin cd -- %q' "$(realpath "$dir")"
```

Comment:
```suggestion
    ) && printf 'builtin cd -- %q' "$(builtin unset CDPATH && builtin cd -- "$dir" && builtin pwd)"
```

Reply:
I think we should revert the extra indentation.

Reply:
ok